docs/monitoring/tag-support.md

```diff
@@ -1,125 +1,113 @@
 
-To collect user defined tags you need to define the IAM roles listed below:
+### Operating System Tags
 
-1. AWS - EC2 Instances should be created with AWS IAM Role that has policy `AmazonEC2ReadOnlyAccess`.
-    See [AWS/EC2 User Guide](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html) for more info.
-2. Azure - To fetch resource tags for Virtual Machines, you need to grant `Reader` role to its Resource Group in Azure Resource Manager
-    See [Access Azure Resource Manager API](https://docs.microsoft.com/en-gb/azure/active-directory/managed-identities-azure-resources/tutorial-linux-vm-access-arm) for more info.
-3. GCE - In GCE User defined tags are called Labels. To read labels, the instance needs `roles/compute.viewer` IAM role.
-    See [Granting Roles to Service Accounts](https://cloud.google.com/iam/docs/granting-roles-to-service-accounts#granting_access_to_a_service_account_for_a_resource) for more info.
+Below are the OS related tags sent as part of OS metrics
 
-Cloud tags collection is enabled by default.  To disable Cloud tags
-collection set `cloud.metadata-enabled` to `false` in `/opt/spm/properties/st-agent.yml` and
-restart spm-monitor using `sudo service spm-monitor restart`.
+| Tag Name  | Description  | Related reserved tags
+|:--|:--|:--
+| os.disk | Human readable name of the block device | 
+| os.disk.mountpoint | Mount point for the disk in the file system |
+| os.disk.fs.type | Type of the file system associated with the device |
+| os.network | Name of the network interface |
 
-#### Machine
+### Process Tags
 
-Following tags are collected from the host the agent is running.
+Below are Process related tags sent as part of metrics/logs process info.
 
-| Name  | Tag Name  | Description |
-|:--|:--|:--|
-| SystemUUID | os.uuid | Unique ID based on SMBIOS specification |
-| OS Distribution Name | os.distro.name | Distribution name of the OS. e.g. `ubuntu` |
-| OS Distribution Version | os.distro.version | Version of the OS. e.g. `16.04` |
-| Kernel Version | os.kernel | Version of the Kernel. e.g. `4.4.0-130-generic` |
-| JVM Version | jvm.version | Version of JVM, if available in `PATH` |
-| Virtualization | virtualization | Virtualization Type. Possible values are `BareMetal`, `VM`, `Container` |
+| Tag Name  | Description  | Related reserved tags
+|:--|:--|:--
+| process.name | process/program name |
+| process.pid  | process identifier | 
 
-### Agent configuration
+### Network Traffic Stats Tags
 
-The Sematext Agents supports configuration of custom tags. They can be specified in the agent's configuration files. For example, you can configure tags like `env:prod` (on all production servers) and `env:dev` (on all dev servers) and filter the data in UI based on these tags. The maximum allowed length for both name and value is 1024 characters. These tags are optional and can be changed anytime.
+Below tags are sent as part of Network Traffic Status metrics:
```

Comment:
Below tags are sent as part of Network Traffic Status metrics:
=>
Below are the tags sent as part of the Network Traffic Status metrics:

Reply:
done